docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.mdx

```diff
@@ -53,35 +58,90 @@ The icon should be somewhat in line with the contemporary icon styles (i.e. not
 
 The name should ideally be no longer than 15 characters, and must be shorter than 20 characters.
 
+:::danger Bad examples
+
+- Apostrophe Markdown
+- Minimal Markdown Editor
+  :::
+
 ### Just the name
 
-The name should be just the name, without any additional info. For example, don't append what the app does to the name (e.g. `Apostrophe - Markdown Editor`), and use the summary to provide this extra information instead.
+The name should be just the name, without any additional info. For example, don't append what the app does to the name, and use the summary to provide this extra information instead.
+
+:::danger Bad examples
+
+- Apostrophe - Markdown Editor
+- Apostrophe (Markdown Editor)
+- Apostrophe (Special Edition)
+  :::
 
 ### No weird formatting
 
 The name should not have any weird formatting or punctuation. For example, it should not be all-lowercase, all-uppercase, camel case, or contain dashes or periods. Cases where the weird formatting is part of the brand are exempt from this (e.g. `VLC`).
 
+:::danger Bad examples
+
+- apostrophe
+- APOSTROPHE
+- ApostropheEditor
+- Apostrophe.org
+  :::
+
 ## Summary
 
 ### Not too long
 
 The summary should ideally be between 10 and 25 characters, and no longer than 35 characters.
 
+:::danger Bad example
+
+- A simple markdown editor for editing files on your system
+  :::
+
 ### Not technical
 
-The summary should be understandable by a non-technical person, and not contain any technical terms (e.g. `Chat app written in Rust and GTK4/Libadwaita`). Don't mention the toolkit, programming language, or other implementation details that would not be relevant to a non-developer and explain what the app does for people instead.
+The summary should be understandable by a non-technical person, and not contain any technical terms. Don't mention the toolkit, programming language, or other implementation details that would not be relevant to a non-developer and explain what the app does for people instead.
+
+:::danger Bad examples
+
+- A GTK4 chat app written in Rust
+- An Electron Wrapper for the Telegram Web App
+- A Qt5 app for managing your steam library
```

Comment:
steam -> Steam?

Reply:
Wondering, if you could think the too technical part is about `Steam`/`Telegram`

Reply:
Depends, but in these cases it might be fine as long as the actual functionality is mentioned somewhere (e.g. if "your Steam games" is mentioned somehow)